server/routes/blog.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const { getABlog, getAllBlog, updateBlog, addNewBlog } = requrie('../controllers/blog.js');
+
+const router = express.Router();
+//  getting all the blogs
+router.get('/blogs', getAllBlog);
+
+//  getting a single blog
+router.get('/blogs', getABlog);
+
+//  posting a new blog
+router.post('/blogs', addNewBlog);
+
+//  updating an existing blog
+router.put('/blogs', updateBlog);
```

Comment:
Need to add a parameter here so we know which blog post is updated. @kalebal does the client have access to the blog post id?

Reply:
Currently just working with dummy data so no, but if the blog post id's are returned on get requests it can.